.github/workflows/post_to_dev_to.yml

```diff
@@ -0,0 +1,30 @@
+---
+name: Post to dev.to
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - master
+
+jobs:
+  post_to_dev_to:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Publish articles on dev.to
+        uses: sinedied/publish-devto@v2
+        with:
+          # Your dev.to personal API key to publish and update articles.
+          # See https://docs.dev.to/api/#section/Authentication/api_key
+          devto_key: ${{ secrets.DEV_TO_TOKEN }}
+          github_token: ${{ secrets.GITHUB_TOKEN }}
+          # (Optional) The files to publish. Default is "posts/**/*.md"
+          files: 'posts/**/2024-*.md'
```

Comment:
only 2024? do we need to update it each year?

Reply:
its for testing, because i do not know if this will release then all of our postings from forever :D